### PR TITLE
doc:add missing components in quickstart;

### PR DESCRIPTION
**Pull request by pape2016**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by ziyoung** [comment]
en-US, fr, es 多语言的更新一下就可以了。

File: examples/docs/zh-CN/quickstart.md [hunk 1]
```diff
@@ -134,12 +134,14 @@ import {
...
   Upload,
   Progress,
+  Spinner,
   Badge,
   Card,
   Rate,
   Steps,
   Step,
   Carousel,
+  Scrollbar,
```
Comment (ziyoung) [inline examples/docs/zh-CN/quickstart.md]: Scrollbar 去掉。

**Comment by pape2016** [comment]
去掉了scrollbar，更新了多语言文档。

File: examples/docs/zh-CN/quickstart.md (hunk 1 above)
Comment (pape2016) [reply examples/docs/zh-CN/quickstart.md]: 去掉了

**Pull request merged**